.github/ISSUE_TEMPLATE/clif-bug-report.md

```diff
@@ -7,13 +7,37 @@ assignees: ''
 
 ---
 
-Thanks for opening a bug report! Please answer the questions below
-if they're relevant and delete this text before submitting.
-
-- What are the steps to reproduce the issue? Can you include a CLIF test case,
-  ideally reduced with the `bugpoint` clif-util command?
-- What do you expect to happen? What does actually happen? Does it panic, and
-  if so, with which assertion?
-- Which Cranelift version / commit hash / branch are you using?
-- If relevant, can you include some extra information about your environment?
-  (Rust version, operating system, architecture...)
+Thanks for filing an issue! Please fill out the TODOs below.
+
+### `.clif` Test Case
+
+```
+TODO: paste .clif test case here. Ideally, a test case that has been reduced via
+the `clif-util bugpoint` command.
+```
+
+### Steps to Reproduce
+
+* TODO: First, ...
+* TODO: Then, ...
+* Etc...
+
+### Expected Results
+
+TODO: What do you expect to happen?
+
+### Actual Results
+
+TODO: What actually happens? Panic? Segfault? Incorrect result?
+
+### Versions and Environment
+
+Cranelift version or commit: TODO
+
+Operating system: TODO
+
+Architecture: TODO
```

Comment:
```suggestion
Cranelift version or commit: e.g., `git describe`?

Operating system: e.g., `uname -osr`

Architecture: e.g., `lscpu | egrep 'Architecture|Model name'`
```

Reply:
I mean, we might want to think up the Windows/MacOS versions of some of those but that would answer @wycats' comments [here](https://github.com/bytecodealliance/wasmtime/issues/2661#issuecomment-780875992).

Reply:
This removes the TODO hints that every other field they need to fill out has, makes the template wordier (which makes it more like an intimidating wall of text), and assumes a unix environment. Not convinced that this change is a clear win.

Reply:
+1 -- OS and architecture would likely be known by folks who would be reporting issues (we can always have a conversation in an issue if not or if we need more detail); whereas complex command pipelines make it look like we're asking for very specific and picky details and may scare folks away, I think.